package.json

```diff
@@ -9,7 +9,7 @@
 		"lint": "eslint . --ext ts,.tsx && tsc",
 		"format": "prettier --write \"**/*.{ts,tsx}\"",
 		"lint:fix": "eslint . --ext ts,.tsx --fix",
-		"pull-manifest": "codegen-graph-ts pull $AELIN_SUBGRAPH_URL > manifest.json",
+		"pull-manifest": "codegen-graph-ts pull https://api.thegraph.com/subgraphs/name/0xlinus/aelin-optimism > manifest.json",
```

Comment:
lets undo this change. I also recommend using your bash_profile or something similar instead of hardcoding for this to make it easier for everyone. we can all manage our own variables however we want